src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String NAME_OF_OPERATION_SUPPLY = "supply";
+    private static final String NAME_OF_OPERATION_BUY = "buy";
+    private static final int INDEX_OF_OPERATION_TYPE = 0;
+    private static final int INDEX_OF_AMOUNT = 1;
+    private static final int INDEX_OF_SUPPLY_AMOUNT = 0;
+    private static final int INDEX_OF_BUY_AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        int[] operationsAmounts = readAmountsOfOperations(fromFileName);
+        supplyAmount = operationsAmounts[INDEX_OF_SUPPLY_AMOUNT];
+        buyAmount = operationsAmounts[INDEX_OF_BUY_AMOUNT];
+
+        String report = createReportMessage(supplyAmount, buyAmount);
+        File fileToWrite = new File(toFileName);
+
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileToWrite))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file", e);
+        }
+    }
```

Comment:
The `getStatistic` method seems to be doing multiple things: reading from a file, creating a report message, and writing to a file. Consider splitting it into smaller methods to adhere to the single responsibility principle.